src/main/java/pl/tomaszdziurko/jvm_bloggers/view/login/LoginPage.java

```diff
@@ -65,11 +75,11 @@ private void tryToLoginUser(String clientAddress) {
 
         CustomFeedbackPanel feedbackPanel = new CustomFeedbackPanel("feedbackPanel");
         loginForm.add(feedbackPanel);
-        RequiredTextField<String> loginField = new RequiredTextField<>(LOGIN_FIELD_ID);
+        RequiredTextField loginField = new RequiredTextField(LOGIN_FIELD_ID);
         loginForm.add(loginField);
         PasswordTextField passwordField = new PasswordTextField(PASSWORD_FIELD_ID);
         loginForm.add(passwordField);
-        Button loginButton =  new Button(FORM_SUBMIT_ID);
+        Button loginButton = new Button(FORM_SUBMIT_ID);
```

Comment:
From my experience this is the hardest thing to learn in code reviews....no golden hammer here.  People are not 0s and 1s ;)